### DISCUSSION TITLE
How to contribute to the documentation?

### DISCUSSION THREAD

**Discussion by vcruzmj** [detekt/detekt#7390] [Q&A] [answered]
I want to contribute to the documentation of the project, but I don't know where to put my stuff, you have a `docs` folder and a `versioned_docs` folder, where do I need to write my stuff?

**Comment by cortinico** [comment, marked as answer]
> but I don't know where to put my stuff, you have a `docs` folder and a `versioned_docs` folder, where do I need to write my stuff?

You need to put it into the `docs` folder at first.
The `versioned_docs` folder is used if you want to make a change that spans across different versions of detekt. 

So I would say let's start from `docs` only. Generally we do a review there and ask you to backport to `versioned_docs` if necessary.
